diff --git a/node_modules/scrollparent/scrollparent.js b/node_modules/scrollparent/scrollparent.js
index fb1088d..2c95341 100644
--- a/node_modules/scrollparent/scrollparent.js
+++ b/node_modules/scrollparent/scrollparent.js
@@ -10,7 +10,7 @@
   var regex = /(auto|scroll)/;
 
   var parents = function (node, ps) {
-    if (node.parentNode === null) { return ps; }
+    if (!node || node.parentNode === null) { return ps; }
 
     return parents(node.parentNode, ps.concat([node]));
   };
@@ -28,9 +28,9 @@
   };
 
   var scrollParent = function (node) {
-    if (!(node instanceof HTMLElement || node instanceof SVGElement)) {
-      return ;
-    }
+    // if (!(node instanceof HTMLElement || node instanceof SVGElement)) {
+    //   return ;
+    // } //
 
     var ps = parents(node.parentNode, []);
 
